src/openms/include/OpenMS/ANALYSIS/QUANTITATION/AbsoluteQuantitationMethod.h

```diff
@@ -106,6 +106,46 @@ namespace OpenMS
     String transformation_model_; ///< transformation model
     Param transformation_model_params_; ///< transformation model parameters
   };
+
+  inline OPENMS_DLLAPI bool operator==(const AbsoluteQuantitationMethod& lhs, const AbsoluteQuantitationMethod& rhs)
+  {
+    if (
```

Comment:
I think you can just use:
return std::tie(lhs.a, , ,  ) == std::tie(rhs.a, ..., ) with all fields you want to compare.
Note that this can also include the Param object (as these also define operator==).
this will simplify this function a lot.